tests/staticfiles_tests/test_storage.py

```diff
@@ -445,6 +445,14 @@ def test_missing_entry(self):
         # File exists on disk
         self.hashed_file_path(missing_file_name)
 
+    def test_doesnt_create_intermediate_files(self):
```

Comment:
The name `test_doesnt_keep_intermediate_files` would be better, to make it come up in a grep for `keep_intermediate_file`.